pkg/glanceapi/deployment.go

```diff
@@ -155,7 +153,7 @@ func Deployment(
 	deployment.Spec.Template.Spec.Affinity = affinity.DistributePods(
 		common.AppSelector,
 		[]string{
-			glance.ServiceName,
+			labels[common.AppSelector],
```

Comment:
-1: decorate the `ObjectMeta` passing:

`Labels: labels` which is already available in the deployment.

Reply:
Sorry but I did not get the suggestion. Do you mind providing the snippet so that I can understand how the code should be fixed ?